samples/cpp/speech_sample/README.md

```diff
@@ -136,7 +136,7 @@ The following pre-trained models are available:
 - rm_lstm4f
 - rm_cnn4a_smbr
 
-All of them can be downloaded from [https://storage.openvinotoolkit.org/models_contrib/speech/2021.2](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2).
+All of them can be downloaded from the [storage](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2).
```

Comment:
All of them can be downloaded from [storage]

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0